core/src/main/resources/error/error-classes.json

```diff
@@ -166,6 +175,9 @@
     "message" : [ "The second argument of '%s' function needs to be an integer." ],
     "sqlState" : "22023"
   },
+  "UDF_WITH_TOO_MANY_TYPE_ARGUMENT_ERROR" : {
+    "message" : [ "UDF class with %s type arguments is not supported" ]
+  },
```

Comment:
Actually, this indicates about a restriction of Spark SQL implementation. Could you use the error class `UNSUPPORTED_FEATURE` and output the error message like
```
The feature is not supported: UDF class with 33 arguments.
```

Reply:
That makes  sense. Fixed.